views/js/qtiCreator/widgets/interactions/extendedTextInteraction/states/Question.js

```diff
@@ -119,16 +132,19 @@ define([
             var response = interaction.getResponseDeclaration();
             var correctResponse = _.values(response.getCorrect());
             var previousFormat = interaction.attr('format');
+            var isMath = attrValue === 'math';
+            var format = isMath ? 'xhtml' : attrValue;
 
             //remove the interaction
             renderer.destroy(interaction);
 
             //change the format and rerender
-            interaction.attr('format', attrValue);
+            interaction.attr('format', format);
+            interaction.attr('data-math-entry', isMath ? 'true' : 'false');
             renderer.render(interaction);
 
-            if(previousFormat === 'xhtml'){
-                if(typeof correctResponse[0] !== 'undefined'){
+            if (format !=='xhtml' && previousFormat === 'xhtml') {
```

Comment:
**chore:** code style issue (I'd suggest activating Prettier to avoid such trivial things)

```suggestion
            if (format !== 'xhtml' && previousFormat === 'xhtml') {
```